examples/express-graphql-http-functions/README.md

```diff
@@ -0,0 +1,15 @@
+# Grats + express + graphql-http
+
+Simple demo project integrating [Grats](https://grats.capt.dev/), [Express](http://expressjs.com/) and [graphql-http](https://graphql-http.com/).
+
+## Concepts used in this example:
+
+* Function-based top level fields
+* How to mimic namespaces in a schema
+* Function based mutations
+* Defining the execution's context type and creating it dynamically per-request
+
+## Running the demo
+* `$ pnpm install`
+* `$ pnpm run build`
+* `$ pnpm run start`
```

Comment:
Can we add a page to our docs for this as well?

https://github.com/captbaritone/grats/tree/main/website/docs/05-examples